libqtile/layout/xmonad.py

```diff
@@ -787,3 +799,227 @@ def cmd_right(self):
+        |___________________|
+        |                   |
+        |                   |
+        |                   |
+        |                   |
+        ---------------------
+
+    Secondary-panes:
+
+    Occupying the rest of the screen are one or more secondary panes.
+    The secondary panes will share the horizontal space of the screen
+    however they can be resized at will with the `cmd_grow' and
+    `cmd_shrink' methods. The other secondary panes will adjust their
+    sizes to smoothly fill all of the space.
+
+        ---------------------          ---------------------
+        |                   |          |                   |
+        |                   |          |                   |
+        |                   |          |                   |
+        |___________________|          |___________________|
+        |     |       |     |          |   |           |   |
+        |     |       |     |          |   |           |   |
+        ---------------------          ---------------------
+
+    Panes can be moved with the `cmd_shuffle_up' and `cmd_shuffle_down'
+    methods. As mentioned the main pane is considered the top of the
+    stack; moving up is counter-clockwise and moving down is clockwise.
+
+    The opposite is true if the layout is "flipped".
+
+        ---------------------          ---------------------
+        |                   |          |  2  |   3   |  4  |
+        |         1         |          |_____|_______|_____|
+        |                   |          |                   |
+        |___________________|          |                   |
+        |     |       |     |          |        1          |
+        |  2  |   3   |  4  |          |                   |
+        ---------------------          ---------------------
+
+    Normalizing:
+
+    To restore all client windows to their default size ratios simply
+    use the `cmd_normalize' method.
+
+
+    Maximizing:
+
+    To toggle a client window between its minimum and maximum sizes
+    simply use the `cmd_maximize' on a focused client.
+
+    Suggested Bindings::
+
+        Key([modkey], "h", lazy.layout.left()),
+        Key([modkey], "l", lazy.layout.right()),
+        Key([modkey], "j", lazy.layout.down()),
+        Key([modkey], "k", lazy.layout.up()),
+        Key([modkey, "shift"], "h", lazy.layout.swap_left()),
+        Key([modkey, "shift"], "l", lazy.layout.swap_right()),
+        Key([modkey, "shift"], "j", lazy.layout.shuffle_down()),
+        Key([modkey, "shift"], "k", lazy.layout.shuffle_up()),
+        Key([modkey], "i", lazy.layout.grow()),
+        Key([modkey], "m", lazy.layout.shrink()),
+        Key([modkey], "n", lazy.layout.normalize()),
+        Key([modkey], "o", lazy.layout.maximize()),
+        Key([modkey, "shift"], "space", lazy.layout.flip()),
+
+    """
+
+    def _get_relative_size_from_absolute(self, absolute_size):
+        return float(absolute_size) / self.group.screen.dwidth
```

Comment:
right, don't know why it's here, perhaps a copy/paste from my old py2.7 layout, sorry for that :-)